Projects/UOContent/Commands/Possess.cs

```diff
@@ -0,0 +1,151 @@
+﻿using System;
+using System.Collections.Generic;
+using Server.Commands.Generic;
+using Server.Mobiles;
+
+namespace Server.Commands
+{
+    internal static class Swap
+    {
+        private static readonly List<Layer> SPECIAL_LAYERS = new() { Layer.Backpack, Layer.Bank, Layer.Mount };
```

Comment:
Do we want to skip the mount and backpack? If we do, firing a weapon or casting a spell will not work right. And a mount is part of possessing some NPCs, right?
We might need to "lock" the player so they can't unmount and "break" the possession identity.